modernpython/templates/cimpy_class_template.mustache

```diff
@@ -22,16 +14,15 @@ class {{class_name}}({{sub_class_of}}):
     """
 
     {{#attributes}}
-    {{^isAssociationUsed}}# *Association not used*
-    # Type {{dataType}} in CIM  # pylint: disable-next=line-too-long
-    # {{/isAssociationUsed}}{{label}} : {{#setType}}{{dataType}}{{/setType}} = Field({{#setDefault}}{{dataType}}{{/setDefault}}, in_profiles = [{{#attr_origin}}Profile.{{origin}}, {{/attr_origin}}]) {{^isAssociationUsed}}# noqa: E501{{/isAssociationUsed}}
-
+    {{#setNormalizedName}}{{label}}{{/setNormalizedName}} : {{#setType}}{{.}}{{/setType}} = Field({{#setDefault}}{{.}}{{/setDefault}}, in_profiles = [{{#attr_origin}}Profile.{{origin}}, {{/attr_origin}}], {{#setCimDataType}}{{.}}{{/setCimDataType}}alias = "{{label}}")
```

Comment:
> The alias worries me a bit.
> 
> Is it needed?

the issue is that several attributes have the same name of the Dataclass they refer to, and pydantic dislike that. (i can reproduce it an report the error)

> 
> Aliases in pydantic are not exactly what I expect them to be. They _have_ to be used (when accessing an attribute). The initial name basically does not exist any more, which makes the type checkers very confused.

https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.populate_by_name

allows to create instances using either the attribute name or the alias. not sure if this works also for accessing...

maybe using validation_alias could work instead of alias, leaving serialisation unchanged? I will test

> 
> With `BaseModel` I am not using them, but if I really need an alias, I create a `computed_field` which is a basically property returning the value of the field I want.
> 
> With `dataclass`, I am not sure how it works.